README.md

```diff
@@ -135,16 +135,17 @@ models:
     ...
     meta:
       fal:
-        scripts:
+        post-hook:
+          # scripts will run concurrently
           - send_slack_message.py
-          - another_python_script.py # will be run after the first script
+          - another_python_script.py
 ```
 
 `fal` also provides useful helpers within the Python context to seamlessly interact with dbt models: `ref("my_dbt_model_name")` will pull a dbt model into your Python script as a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
 
-### Running scripts before dbt runs
+## Running scripts before dbt runs
 
-The `--before` flag let's users run scripts before their dbt runs.
+By using `pre-hook` you can run scripts before the associated models run.
```

Comment:
Run scripts before the model runs by using the `pre-hook:` configuration option.